client/src/components/history/history-item.tsx

```diff
@@ -50,13 +51,15 @@ export function ChatItem(props: {
   styles: StyleProps;
   setAnswerVisibility: (show: boolean) => void;
   visibility: boolean;
+  config: Config;
 }): JSX.Element {
-  const { message, i, styles, setAnswerVisibility, visibility } = props;
+  const { message, i, styles, setAnswerVisibility, visibility, config } = props;
   const [anchorEl, setAnchorEl] = React.useState<Element | null>(null);
   const dispatch = useDispatch();
   const isUser = !message.mentorId;
 
   const mentorColor = message.color || "#88929e";
+  const prefix = "question://";
```

Comment:
I think Ben Nye wanted this to be `ask://`

Reply:
OK, changed